System/Posix/Time.hsc

```diff
@@ -39,3 +54,92 @@ epochTime = throwErrnoIfMinus1 "epochTime" (c_time nullPtr)
 
 foreign import ccall unsafe "__hsunix_time"
   c_time :: Ptr CTime -> IO CTime
+
+#ifdef HAVE_SETITIMER
```

Comment:
Which still supported POSIX platforms don't have setitimer?  Likely the `#if` guards are long redundant.